src/Sentry/Protocol/StructuredLog.cs

```diff
@@ -0,0 +1,42 @@
+using Sentry.Extensibility;
+
+namespace Sentry.Protocol;
+
+/// <summary>
+/// Represents the Sentry Log protocol.
+/// </summary>
+/// <remarks>
+/// Sentry Docs: <see href="https://docs.sentry.io/product/explore/logs/"/>.
+/// Sentry Developer Documentation: <see href="https://develop.sentry.dev/sdk/telemetry/logs/"/>.
+/// </remarks>
+internal sealed class StructuredLog : ISentryJsonSerializable
+{
+    private readonly SentryLog[] _items;
+
+    public StructuredLog(SentryLog log)
```

Comment:
can we get rid of this? do we use this constructor? If we anyway allocate the array, just keep this simple and let the caller do that. It's an internal API. 

Less is more